README.md

```diff
@@ -21,7 +21,9 @@ Check the [releases page](https://github.com/IDPF/epubcheck/releases) to get the
 
 Documentation on how to **use** EpubCheck, to **contribute** to the project or to **translate** messages is available on the [EpubCheck wiki](https://github.com/IDPF/epubcheck/wiki).
 
-Technical discussions are hosted on the [EpubCheck Google Group](https://groups.google.com/forum/#!forum/epubcheck)
+Technical discussions are hosted are held on our public [email list](mailto:'public-epubcheck@w3.org'). To subscribe to the email list, send an email with subject "subscribe" to [public-epubcheck@w3.org](mailto:'public-epubcheck@w3.org'). To participate in the discussion, simply send an email to [public-epubcheck@w3.org](mailto:'public-epubcheck@w3.org'). 
```

Comment:
I think the `mailto:` email address must not be enclosed by single quotes.

Reply:
Link "email list" to the list page at https://lists.w3.org/Archives/Public/public-epubcheck/ ?

Reply:
P.S.: You can omit the link syntax for Markdown and just write the email address as is, this should render as a link on GitHub.

Reply:
The correct address to use _for subscription_ is public-epubcheck-request@w3.org. Alternatively, go to the archive of the mailing list (https://lists.w3.org/Archives/Public/public-epubcheck/) and click on the relevant button. Adding a reference to the archive is also a good idea anyway, imho.